cpp/ql/src/experimental/Best Practices/WrongUintAccess.ql

```diff
@@ -0,0 +1,27 @@
+/**
+ * @id cpp/wrong-uint-access
+ * @name Wrong Uint
+ * @descripion Acess an array of size lower than 256 with a uint16.
+ * @kind problem
+ * @problem.severity recommendation
+ * @tags efficiency
+ */
+
+import cpp
+
+from
+  Variable var, ArrayExpr useExpr, VariableDeclarationEntry def, ArrayType defLine,
```

Comment:
I don't think you're using `def` anymore now, right?

Reply:
Yes it's not used again. I forgot to remove it, but it's done.